src/webgl/p5.Geometry.js

```diff
@@ -2010,11 +2010,10 @@ p5.Geometry = class Geometry {
  * }
  * </code>
  * </div>
-/
-/**
- * @method setAttribute
+ *
  * @param {String} attributeName the name of the vertex attribute.
  * @param {Number|Number[]} data the data tied to the vertex attribute.
+ * @param {Number} size optional size of each unit of data
```

Comment:
We can mark a parameter as optional in the friendly error system's validator too by putting its name in square brackets in the doc comments:

```suggestion
 * @param {Number} [size] optional size of each unit of data
```